pylint/checkers/variables.py

```diff
@@ -391,6 +391,11 @@ def _has_locals_call_after_node(stmt, scope):
         "invalid-all-object",
         "Used when an invalid (non-string) object occurs in __all__.",
     ),
+    "E0605": (
+        "Invalid format for __all__, must be tuple or list",
+        "invalid-all-format",
+        "Used when __all__ has an invalid format.",
```

Comment:
```suggestion
        "Used when __all__ has is neither a tuple nor a list.",
```

Reply:
I don't think that is necessary. The documentation also shows the message which already contains the info about `tuple` and `list`.

https://pylint--4537.org.readthedocs.build/en/4537/technical_reference/features.html#variables-checker-messages